src/use-cases/get-metadata.ts

```diff
@@ -0,0 +1,14 @@
+import S3Gateway from '../gateways/S3-Gateway';
+
+class GetMetadata {
+  s3Gateway: S3Gateway;
+  constructor(s3Gateway: S3Gateway) {
+    this.s3Gateway = s3Gateway;
+  }
+
+  async execute(documentId: string): Promise<{ [key: string]: string }> {
```

Comment:
In my PR I added an interface for the "metadata" type - I'm cool with merging this and I'll update to use that when I merge my PR in.